src/AutoRest.CSharp/Mgmt/Models/MgmtClientOperation.cs

```diff
@@ -209,6 +198,36 @@ private FormattableString BuildDescription()
             return parameterMappings;
         }
 
+        private MethodSignature EnsureMethodSignature()
+        {
+            var attris = this._operations
```

Comment:
we really do not need to use a short form here, please just use `attributes` it is not really long.
```suggestion
            var attributes = this._operations
```